src/components/Button.js

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ buttonName }) => <button type="button">{buttonName}</button>;
-
+const Button = ({ buttonName, color, wide }) => {
+  const twice = wide === true && 'wide';
+  return (
+    <button type="button" className={`button border ${twice} ${color}`}>
```

Comment:
I think the `twice` variable is redundant.

If wide is `false`, what do you want the `twice` variable to be (currently it will be `false`)?

Let's add this condition directly as a prop where twice is used, that is, something like `${wide ? 'wide' : ''}` will do the same thing `${twice}` is doing.

Reply:
I'll work on this